packages/devtools/plugin/src/runtime.ts

```diff
@@ -54,10 +82,12 @@ const setup = () => {
   const cookieManifestUrl = document.cookie.match(
     /use_modernjs_devtools=([^;]+)/,
   )?.[1];
-  const devtoolsUrl = cookieManifestUrl || injectedManifestUrl;
+  const devtoolsUrl = injectedManifestUrl || cookieManifestUrl;
```

Comment:
What is the reason for the logical change here?

Reply:
For some cases developers may serve both development and production server at a same domain.  For the previous implementation, if people setup a devtools cookie for production service, it will also affect the development server. So it has being changed to use the manifest url injected by bundler in primary.